index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
+      </div>
+    </div>
+    <!-----------Header section------------->
+    <header>
+      <div class="main-header__brand">
+        <a href="index.html">
+          <img src="images/logo.png" alt="Logo" />
+        </a>
+      </div>
+    </header>
+
+    <section class="main-section">
+      <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
```

Comment:
- [ ]  Kindly ensure that the color, font size, line height, and font weight of the introductory text match the Figma design you chose as required by the previous review [here](https://github.com/akhror-valiev/micronauts-portfolio/pull/1/files/6ca7b69ff39513d968726a40c01ba92380660e37#r977257767 ) . You can follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to the Figma design specification for the exact properties used👍🏽